packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -107,6 +107,33 @@ export const TerminalConfigSchema: PreferenceSchema = {
             type: 'number',
             default: 1
         },
+        'terminal.integrated.shell.windows': {
+            type: 'string',
+            description: 'The path of the shell that the terminal uses on Windows (default: C:\\Windows\\System32\\cmd.exe).',
+            default: 'C:\\Windows\\System32\\cmd.exe'
```

Comment:
@owlJaeger should we align the defaults similarly to vscode?:

https://github.com/microsoft/vscode/blob/df1f0712f6f8def6e0b52065b6fe66dab66c4e53/src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts#L330-L353

```ts
'terminal.integrated.shell.linux': {
  markdownDescription:
    getSystemShell
	? localize('terminal.integrated.shell.linux', "The path of the shell that the terminal uses on Linux (default: {0}). [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration).", getSystemShell(Platform.Linux))
    : localize('terminal.integrated.shell.linux.noDefault', "The path of the shell that the terminal uses on Linux. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration)."),
  type: ['string', 'null'],
  default: null
},
'terminal.integrated.shell.osx': {
  markdownDescription:
    getSystemShell
	? localize('terminal.integrated.shell.osx', "The path of the shell that the terminal uses on macOS (default: {0}). [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration).", getSystemShell(Platform.Mac))
	: localize('terminal.integrated.shell.osx.noDefault', "The path of the shell that the terminal uses on macOS. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration)."),
  type: ['string', 'null'],
  default: null
},
'terminal.integrated.shell.windows': {
  markdownDescription:
    getSystemShell
	? localize('terminal.integrated.shell.windows', "The path of the shell that the terminal uses on Windows (default: {0}). [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration).", getSystemShell(Platform.Windows))
	: localize('terminal.integrated.shell.windows.noDefault', "The path of the shell that the terminal uses on Windows. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration)."),
  type: ['string', 'null'],
  default: null
}
```

We want to support the same `type` so that the preferences are compatible with vscode.
The framework is able to successfully pick up a workspace's `.vscode/settings.json` and we want to make sure to align.

Reply:
I tested it and setting the `terminal.integrated.shell.windows` property to powershell, changes the shell Theia uses. Maybe it should be tested on other systems too.

Reply:
The defaults still look inconsistent, is there a reason not to use them?

Reply:
Does this look better to you?

Reply:
These new changes include `type: ['string', 'null']` & `default: undefined`.

Reply:
I now see this as resolved. If other concerns arise, please reopen.